core/executor/src/lib.rs

```diff
@@ -94,14 +94,13 @@ impl Executor for AxonExecutor {
         let mut hashes = Vec::with_capacity(txs_len);
         let mut gas_use = 0u64;
 
-        let evm_executor = EvmExecutor::new();
-        let sys_executor = SystemExecutor::new();
-
```

Comment:
Why change this?

Reply:
because different executor is needed under different situations, no need to always create both.  
just to improve this by the way